frontend_tests/node_tests/upload.js

```diff
@@ -62,24 +62,27 @@ run_test("get_item", () => {
         $("#undo_markdown_preview"),
     );
 
-    assert.equal(upload.get_item("textarea", {mode: "edit", row: 1}), $("#message_edit_content_1"));
+    assert.equal(
+        upload.get_item("textarea", {mode: "edit", row: 1}),
+        $(`#message_edit_content_${CSS.escape(1)}`),
+    );
 
-    $("#message_edit_content_2").closest = () => {
+    $(`#message_edit_content_${CSS.escape(2)}`).closest = () => {
         $("#message_edit_form").set_find_results(".message_edit_save", $(".message_edit_save"));
         return $("#message_edit_form");
     };
     assert.equal(upload.get_item("send_button", {mode: "edit", row: 2}), $(".message_edit_save"));
 
     assert.equal(
         upload.get_item("send_status_identifier", {mode: "edit", row: 11}),
-        "#message-edit-send-status-11",
+        `#message-edit-send-status-${CSS.escape(11)}`,
     );
     assert.equal(
         upload.get_item("send_status", {mode: "edit", row: 75}),
-        $("#message-edit-send-status-75"),
+        $(`#message-edit-send-status-${CSS.escape(75)}`),
```

Comment:
This is another place where `CSS.escape` feels like clutter to me.

Reply:
See [above](https://github.com/zulip/zulip/pull/17200#discussion_r570444414).